hedera-node/src/main/java/com/hedera/services/grpc/NettyGrpcServerManager.java

```diff
@@ -73,14 +81,29 @@ public void start(int port, int tlsPort, Consumer<String> println) {
 		}
 	}
 
-	private Server startOneNettyServer(boolean sslEnabled, int port, Consumer<String> println) throws Exception {
+	Server startOneNettyServer(boolean sslEnabled, int port, Consumer<String> println, Pause pause) throws Exception {
 		println.accept(nettyAction("Starting", sslEnabled, port, true));
 
 		NettyServerBuilder builder = nettyBuilder.builderFor(port, sslEnabled);
 		bindableServices.forEach(builder::addService);
 		serviceDefinitions.forEach(builder::addService);
 		Server server = builder.build();
-		server.start();
+
+		var retryNo = 1;
+		final var n = Math.max(0, nodeProperties.nettyStartRetries());
+		for (; retryNo <= n; retryNo++) {
+			try {
+				server.start();
+				break;
+			} catch (IOException e) {
+				final var summaryMsg = nettyAction("Still trying to start", sslEnabled, port, true);
+				log.warn("(Attempts=" + retryNo + ") " + summaryMsg + e.getMessage());
+				pause.forMs(nodeProperties.nettyStartRetryIntervalMs());
+			}
+		}
+		if (retryNo == n + 1) {
```

Comment:
Another nit: It's always safer doing `retryNo > n` than this strict expectation. Also, I don't know why Codecov complains about the `server.start();` in this if, but you did cover it in the `givesUpIfMaxRetriesExhaustedAndPropagatesIOException` test.